subprojects/common/elastic/src/main/kotlin/com/avito/android/elastic/RoundRobinIterable.kt

```diff
@@ -0,0 +1,13 @@
+package com.avito.android.elastic
+
+internal class RoundRobinIterable<out T>(private val collection: Iterable<T>) {
```

Comment:
Do we really do load balancing on the client-side? Or is it just "send to all"?

Reply:
yes it's strange client side balancing for now

Reply:
i wrote small note about it in class kdoc